src/screens/CreateNFT.js

```diff
@@ -0,0 +1,333 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import $ from 'jquery';
+import { t } from 'ttag'
+import { str2jsx } from '../utils/i18n';
+
+import wallet from '../utils/wallet';
+import tokens from '../utils/tokens';
+import SpanFmt from '../components/SpanFmt';
+import ModalSendTx from '../components/ModalSendTx';
+import ModalAlert from '../components/ModalAlert';
+import { connect } from "react-redux";
+import BackButton from '../components/BackButton';
+import hathorLib from '@hathor/wallet-lib';
+import helpers from '../utils/helpers';
+import { NFT_STANDARD_RFC_URL } from '../constants';
+import InputNumber from '../components/InputNumber';
+
+
+const mapStateToProps = (state) => {
+  const HTR_UID = hathorLib.constants.HATHOR_TOKEN_CONFIG.uid;
+  const htrBalance = HTR_UID in state.tokensBalance ? state.tokensBalance[HTR_UID].available : 0;
+  return {
+    htrBalance,
+    wallet: state.wallet,
+  };
+};
+
+
+/**
+ * Create an NFT
+ *
+ * @memberof Screens
+ */
+class CreateNFT extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.address = React.createRef();
+    this.inputWrapper = React.createRef();
+
+    /**
+     * errorMessage {string} Message to show when error happens on the form
+     * name {string} Name of the created token
+     * configurationString {string} Configuration string of the created token
+     * amount {number} Amount of tokens to create
+     */
+    this.state = {
+      errorMessage: '',
+      name: '',
+      configurationString: '',
+      amount: null,
+    };
+  }
+
+  /**
+   * Validates if the create NFT form is valid
+   */
+  formValid = () => {
+    const isValid = this.refs.formCreateNFT.checkValidity();
+    if (isValid) {
+      if (this.refs.address.value === '' && !this.refs.autoselectAddress.checked) {
```

Comment:
`refs` was [deprecated](https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs).

Reply:
Refactor done at https://github.com/HathorNetwork/hathor-wallet/pull/222/commits/e8bccac102b5d57f7bc6f864d29d7390e8f4d69e